docs/CONFIG-VARS.md

```diff
@@ -155,6 +156,13 @@ When V4_CFG_MANAGE_STORAGE is set to `true`, the `sas` and `pg-storage` storage
 | V4_CFG_INGRESS_FQDN | FQDN to the ingress for SAS Vya installation | string | | true | | viya |
 | V4_CFG_INGRESS_MODE | Whether to create a public or private Loadbalancer endpoint | string | "public" | false | Possible values: "public", "private". Setting this option to "private" adds options to the ingress controller that create a LoadBalancer with private IP address(es) only. | baseline |
 
+## Load Balancer
+
+| Name | Description | Type | Default | Required | Notes | Tasks |
+| :--- | ---: | ---: | ---: | ---: | ---: | ---: |
+| V4_CFG_AWS_LB_SUBNETS | The AWS subnets and by association the AWS availability zones to deploy the load balancing service to. Specifying a subnet value or values for this variable takes precedence over the Subnet Discovery method described in [AWS docs](https://docs.aws.amazon.com/eks/latest/userguide/network-load-balancing.html) and eliminates the need to tag AWS subnets as documented in scenario 2 of this [table](https://github.com/sassoftware/viya4-iac-aws/blob/main/docs/user/BYOnetwork.md#supported-scenarios-and-requirements-for-using-existing-network-resources) | string | | false | The value is either a comma separated list of subnet IDs, or a comma separated list of subnet names. | baseline |
+
+
```

Comment:
Is this not a BYO networking type item? Seems like it belongs in another location.

Reply:
It is most likely a BYO networking concern although it is not limited to just some BYO networking scenario. Can also be used with BYON=0 which equates to not bringing your own network. I put it under its own heading as I didn't think it belonged in any existing section, I will see if I can find an alternative to suggest.

Reply:
@thpang, we don't actually have any existing BYO doc in the DAC repo (the link I have in that section points back to the AWS repo's BYOnetwork.md doc, so it's all back in each IAC project currently.

Reply:
I am going to add to the description that this variable can be used with any BYO networking scenario 0-3.